### ISSUE TITLE
[Visual bug/LogiX] LogiX node inputs and references have weird lines

### ISSUE THREAD

**Issue by 3x1t-5tyl3** [Frooxius/NeosPublic#1093]
Generally LogiX nodes and in/-outputs have weird lines (see screenshot below) which can easily be fixed by switching the sprite texture WrapModeU/-V from "Repeat" to "MirrorOnce".
![20200928142415_1](https://user-images.githubusercontent.com/61988002/94432075-7e436400-0196-11eb-9121-4be69356a401.jpg)
They are particularly noticeable on low resolution because of AntiaAliasing.
The reason this happens is because it grabs the first line of the (triangle) of pixels and adds them to the end instead of terminating at the end of it. (I assume this wrap mode is inteded behaviour for better mirroring?)

**Comment by Frooxius** [comment]
Yes, this is a known issue, but hasn't been prioritized. The textures need to be switched to Clamped in a bunch of different places to avoid these artifacts. The whole LogiX visuals are planned to be reworked at some point in the future, so it might not really get fixed for this version since it doesn't technically break anything.
